### PR TITLE
clipping sea ice tendencies

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

In Holloway and Manabe 1971 it says

> If the solution to this heat balance equation results in a value of T* above freezing, this value is set back to freezing with the implicit assumption that the excess heat is used for melting pack ice.

The current code sets the tendency to zero if it results in T above freezing, which differs from the paper. I change it to clip the tendency to (T_freeze - T) / dt, which should match the paper more closely. I also remove a comment about sublimation which doesn't seem to be true in the code.

Edit: One out of four simulations break with dt 240 in the latest nightly build: https://buildkite.com/clima/climacoupler-coarse-nightly-amip/builds/73. So I change dt back to 180 in this PR, and change t_end to one and a half years.

## To-do

I will revert the change in nightly pipeline before merging.

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Review by juliasloan25 — APPROVED** [review]

File: .buildkite/nightly/pipeline.yml [hunk 1]
```diff
@@ -65,47 +65,47 @@ steps:
...
+      #     - "julia --threads=3 --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file $CONFIG_PATH/amip_coarse.yml --job_id amip_coarse4"
+      #   artifact_paths: "experiments/ClimaEarth/output/amip/amip_coarse4_artifacts/*"
+      #   timeout_in_minutes: 1080
+      #   env:
+      #     CLIMACOMMS_DEVICE: "CUDA"
+      #   agents:
+      #     slurm_gpus_per_task: 1
+      #     slurm_cpus_per_task: 4
+      #     slurm_ntasks: 1
+      #     slurm_mem: 30GB
```
Comment (Sbozzolo) [inline .buildkite/nightly/pipeline.yml:111]: Do you want these commented out?

File: experiments/ClimaEarth/components/ocean/prescr_seaice.jl [hunk 2]
```diff
@@ -174,12 +172,11 @@ function ice_rhs!(du, u, p, _)
...
-
-    # do not count tendencies that lead to temperatures above freezing, and mask out no-ice areas
+    # If tendencies lead to temperature above freezing, set temperature to freezing
+    @. rhs = min(rhs, (T_freeze - Y.T_sfc) / p.dt)
+    # mask out no-ice areas
     area_mask = Regridder.binary_mask.(area_fraction)
-    threshold = zero(FT)
-    unphysical = @. Regridder.binary_mask.(T_freeze - (Y.T_sfc + FT(rhs) * FT(p.dt)), threshold) .* area_mask
-    parent(dY.T_sfc) .= parent(rhs .* unphysical)
+    parent(dY.T_sfc) .= parent(rhs .* area_mask)
```
Comment (Sbozzolo) [inline experiments/ClimaEarth/components/ocean/prescr_seaice.jl:179]: Would this work?

```suggestion
    dY.T_sfc .= rhs .* area_mask
```

File: .buildkite/nightly/pipeline.yml (hunk 1 above)
Comment (szy21) [reply .buildkite/nightly/pipeline.yml:111]: No, I will revert this before merging.

File: experiments/ClimaEarth/components/ocean/prescr_seaice.jl (hunk 2 above)
Comment (szy21) [reply experiments/ClimaEarth/components/ocean/prescr_seaice.jl:179]: I think there might be a reason why we have parent before, but let me try.

Comment (szy21) [reply experiments/ClimaEarth/components/ocean/prescr_seaice.jl]: it works:)

**Comment by szy21** [comment]
Update: One out of four simulations break with dt 240 in the latest nightly build: https://buildkite.com/clima/climacoupler-coarse-nightly-amip/builds/73. So I change dt back to 180 in this PR, and change t_end to one and a half years.

File: experiments/ClimaEarth/components/ocean/prescr_seaice.jl (hunk 2 above)
Comment (Sbozzolo) [reply experiments/ClimaEarth/components/ocean/prescr_seaice.jl]: I suspected so

**Pull request merged**